index.html

```diff
@@ -0,0 +1,384 @@
+                    Julian Jameson
+                </label>
+                <label class="team-card__text team-card__text--muted">
+                    Proffesion
+                </label>
+                <div class="team-card__socials">
+                    <img class="team-card__icon" src="./img/fb.png">
+                    <img class="team-card__icon" src="./img/ig.png">
+                    <img class="team-card__icon" src="./img/tw.png">
+                </div>
+            </div>
+            <div class="team-card">
+                <img class="team-card__image" src="./img/user-3.png">
+                <label class="team-card__text team-card__text--bold">
+                    Julian Jameson
+                </label>
+                <label class="team-card__text team-card__text--muted">
+                    Proffesion
+                </label>
+                <div class="team-card__socials">
+                    <img class="team-card__icon" src="./img/fb.png">
+                    <img class="team-card__icon" src="./img/ig.png">
+                    <img class="team-card__icon" src="./img/tw.png">
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="appointment">
+        <div class="appointment__subtitle">Contact us</div>
+        <div class="  title appointment__title">Make an Appointment</div>
+        <form class="form">
+            <input class="form__input" type="text" placeholder="full name">
+            <input class="form__input" type="text" placeholder="example@email.com">
+            <input class="form__input" type="text" placeholder="Please Select">
+            <input class="form__input" type="text" placeholder="4:00 Available">
+            <input class="form__message" type="text">
+
+        </form>
+        <button class="button button--block">
+            Book Appointment
+        </button>
+    </div>
+    <div class="consultation" >
+        <div class="consultation__body">
+            <div class="consultation__title">
+                Consulting Agency 
+                For Your Business
+            </div>
+            <div class="description description--justify-left description--not-padded">
+                the quick fox jumps over 
+                the lazy dog
+            </div>
+        </div>
+        <button class="button--block button">Contact Us</button>
+    </div>
+    <div class="footer">
+        <div class="footer__socials">
+            <div class="footer__logo">KMarketer</div>
+            <div class="footer__icons">
+                <img src="./img/fb-blue.png">
+                <img src="./img/ig red.png">
+                <img src="./img/tw.png">
+                <img src="./img/yt.png">
+            </div>
+        </div>
+        <div class="footer__links">
+            <div class="links">
+                <div class="links__header">Company Info</div>
+                <div class="">About Us</div>
```

Comment:
Why did you leave this class empty for these elements?

Reply:
I corrected it and the previous mistakes too.